From 18f5ca42b6cfba7e1155803da4f2767bf461cfcb Mon Sep 17 00:00:00 2001
From: Triet Nguyen <triet.nguyen.te@renesas.com>
Date: Wed, 8 Sep 2021 09:29:28 +0700
Subject: [PATCH] v4l2object: Update min buffer of CRU module for RZ/G2L

This configure set min buffer to ensure min buffer
satisfy with continuous frame capture mode of RZ/G2L CRU driver.
CRU driver can support continuous capture frame with 8 or more buffers.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 sys/v4l2/gstv4l2object.c | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/sys/v4l2/gstv4l2object.c b/sys/v4l2/gstv4l2object.c
index d1aa1be..49212fb 100644
--- a/sys/v4l2/gstv4l2object.c
+++ b/sys/v4l2/gstv4l2object.c
@@ -863,11 +863,10 @@ gst_v4l2_get_driver_min_buffers (GstV4l2Object * v4l2object)
   }
   if (GST_IS_V4L2SRC (v4l2object->element) == TRUE) {
 #ifdef CONT_FRAME_CAPTURE
-    /* RCarVIN driver support 2 transmission modes:
-     * Single frame capture mode: require 3 or less buffers
-     * Continuous frame capture mode: require 4 or more buffers
+    /* RZ/G2L driver support Continuous frame capture transmission modes:
+     * Continuous frame capture mode: require 8 or more buffers
      */
-    v4l2object->min_buffers = 4;
+    v4l2object->min_buffers = 8;
 #endif
   }
 }
-- 
2.17.1

